modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -45,6 +45,7 @@
             <class name="org.wso2.identity.integration.test.application.mgt.ImportExportServiceProviderTest"/>
             <class name="org.wso2.identity.integration.test.application.mgt.ApplicationTemplateMgtTestCase"/>
             <!--<class name="org.wso2.identity.integration.test.application.mgt.DefaultAuthSeqManagementTestCase"/>-->
+            <class name="org.wso2.identity.integration.test.application.mgt.ServiceProviderUserRoleValidationTestCase"/>
```

Comment:
As this test case have a configuration change, we need to add this to `is-tests-with-individual-configuration-changes` test suite.  https://github.com/wso2/product-is/blob/0688b361ec4ad514a67ed6848e2b2e15cf6c8a26/modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml#L306

Reply:
Fixed with https://github.com/wso2/product-is/pull/13308/commits/1d02ae0e5259cfa0048206403bf8503c61b05387.